packages/fxa-content-server/app/styles/_layout.scss

```diff
@@ -89,6 +89,11 @@
   margin-top: 24px;
 }
 
+#fxa-404-p {
```

Comment:
Much better! Although, we've actually got another issue setup around not using IDs to style, we should use a class instead. Would you mind changing this to a class (`.fxa-404-p`) instead of an ID?

Also, there's a couple of things around your commit before this can be merged. Check out [our contributing guidelines](https://github.com/mozilla/fxa/blob/main/CONTRIBUTING.md#sending-pull-requests) but the TLDR to get this PR merged is to please:
* Make sure your commit is [GPG signed](https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification)
* Squash your commits into one, you can use `git rebase -i main` to interactively "fix up" the second commit and "reword" the first commit
* Use the format suggested for your commit, such as:
```
fix(style): Update 404 pages paragraph styles

Because:
* The line-height on paragraphs on 404 pages did not match the line-height on other FxA pages.

This commit:
* Adds a class on the 404 page `p` tag and adds styles.
```

Doesn't have to match my wording but we've found using this format consistently helps provide a lot of context when using `git blame` or `git log`. 👍 Let me know if you've got any questions.

Reply:
Thank you. I've made the change and have signed the Commit.

Reply:
@SrilalS thanks for making those changes! PR looks good to me. Hmm, although it looks like your commit signature is still showing as `unverified`. Will you double check that's set correctly? Additionally, we should only see one commit message here. If you run `git rebase -i main`, you should be able to mark the last 3 commits as "f" or "fix up". When rebased it'll merge into one new commit, and then you can force-push to this branch.

Reply:
@LZoog Thank you for your help. I have rebased and fixed the unverified issue and force pushed it.

Reply:
@SrilalS no problem, thanks for sticking with this PR! Sorry to have even more back and forth here. 😓

Looks like you were able to successfully squash/fixup your commits, but I'm still not seeing that the commit signature is GPG signed - typically if you go to the "commits" tab you'll see a "verified" note next to the commit, here's a screenshot of one of mine: 
<img width="327" alt="image" src="https://user-images.githubusercontent.com/13018240/90036575-d37bf280-dc88-11ea-877f-9ea013e339a2.png">
If you run `git config --list` do you see `commit.gpgsign=true`? If you run `git verify-commit 432e80cc` do you see a "Signature made using RSA key [key] with a "Good signature from [you]" listed?

We also have a [commit message format](https://github.com/mozilla/fxa/blob/main/CONTRIBUTING.md#git-commit-guidelines) to follow, I'd pasted an example above under "Use the format suggested for your commit, such as:". While the "because" and "this commit" sections can help give more context, the required part is that your commit start with `<type>(<scope>):`, such as `fix(style):`. When we run our release script, it runs through all new commits from our previous release to create our changelog files and this format helps with sorting. After the GPG signature is sorted, reword your commit to start with this and force-push if needed, and we'll be good to go. 👌

Reply:
@LZoog no problem. as a student I'm learning a lot here. I have verified the commit and changed the commit message according to the commit message format.

Reply:
@SrilalS So glad you were able to figure it out, thanks again for helping out here!

Reply:
@LZoog Thank you for your Support.